### PR TITLE
Added content to the troubleshoot page.

### PR DESCRIPTION
**Pull request by MRCattos**
### Type
- [X] Rewrite already written documentation
- [ ] Add/remove new pages

### Questions
- [X] Did you preview on your local machine beforehand?

### Describe your PR
This fork updates the Troubleshooting page on the Atlas OS website with some of the most frequently submitted issue and their potential fixes.

### CODE REVIEW

**Comment by MRCattos** [comment]
I am also planning on adding possible fixes for: docker/hyper -v not working, microsoft store not working and other stuff that i can find on the forums that gets requested alot.

File: docs/troubleshooting/index.md [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+
+## No image preview ?
+1. Enable animations from atlas folder which is located at: `Atlas/3. Configuration/1. General Configuration/Animations`
+2. Press the windows key and type "adjust the appearance and performance of windows" and enable "show thumbnails insteaf of icons".
+3. Make sure "Always show icons, never thumbnails" is off in the file explorer.
+
+<sup>Credits 1-2: 𝚍𝚢𝚗𝚊𝚖𝚘#2237 </sup><br>
+<sup>Credits 3: Legend-FY#8603 </sup>
+
+### Want to contribute ?
```
Comment (PencilNavigator) [inline docs/troubleshooting/index.md:39]: um not very nesscery, we have that.

File: docs/troubleshooting/index.md [hunk 2]
```diff
@@ -1,3 +1,40 @@
...
+<sup>Credits: Styris#2225 </sup>
+
+Still stuck? Feel free to make a forum post in the Atlas Discord server.
+
+## No image preview ?
+1. Enable animations from atlas folder which is located at: `Atlas/3. Configuration/1. General Configuration/Animations`
+2. Press the windows key and type "adjust the appearance and performance of windows" and enable "show thumbnails insteaf of icons".
+3. Make sure "Always show icons, never thumbnails" is off in the file explorer.
+
+<sup>Credits 1-2: 𝚍𝚢𝚗𝚊𝚖𝚘#2237 </sup><br>
```
Comment (PencilNavigator) [inline docs/troubleshooting/index.md:36]: your github profile will be shown in the docs page, if you have contributed to the file. 
https://docs.atlasos.net/ automatically upates with the latest contributor of the file.

图片

File: docs/troubleshooting/index.md [hunk 3]
```diff
@@ -1,3 +1,40 @@
...
\ No newline at end of file
+## Password not working
+After the second restart when completing the the ame wizzard some people need a password to log into their account. <br />
+Try **leaving the password empty or type in "atlas"** to see if that works. <br />
+If that doesn't work feel free to make a post in the Atlas forum section on the Atlas Discord server.
+
+## Reticulating Splines
+After running the playbook with the ame wizzard some people get stuck at the "Recticulating Splines" screen. <br/>
+### First possbible fix:
+Check if you have **both the ame wizzard and the playbook on your dekstop**. If not, please stop the ame wizzard with taskmanager, move both the wizzard and playbook to your desktop and try again.
```
Comment (PencilNavigator) [inline docs/troubleshooting/index.md:13]: spelling mistake "wizzard"

**Review by PencilNavigator — APPROVED** [review]
Overall pretty well done! Just some small changes and wait for another person to review and will be good to merge

**Comment by JayXTQ** [comment]
@PencilNavigator I think I have reformatted correctly, take a peak.

**Comment by JayXTQ** [comment]
also @MRCattos please do not mark things as resolved, unless they are ofc.

**Comment by JayXTQ** [comment]
@PencilNavigator why reopen?

**Comment by PencilNavigator** [comment]
@JayXTQ check if this is ok

File: docs/troubleshooting/common-issues/hyper-v.md
Comment (JayXTQ) [inline docs/troubleshooting/common-issues/hyper-v.md:1]: Hyper-V can just be enabled via the script no? No need to do all of this. Also this is already in the docs to do with the script

Comment (MRCattos) [reply docs/troubleshooting/common-issues/hyper-v.md:1]: Did a fresh install yesterday on a vm and the Hyper-V script didn't work

Comment (JayXTQ) [reply docs/troubleshooting/common-issues/hyper-v.md:1]: @Xyueta can you confirm the hyper-v script is broken and this is the fix?

**Comment by Xyueta** [comment]
I added only the required changes in https://github.com/Atlas-OS/docs/commit/d2ccfba9feece51c50004f56b42c4ede7ad15501.

Other things are fixed now or are going to be fixed, so there is not actual need to add them.

Thanks though.

**Pull request closed without merging**